plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src/org/wso2/developerstudio/eclipse/gmf/esb/provider/InboundEndpointItemProvider.java

```diff
@@ -2513,6 +2520,160 @@ protected void addStatisticsEnabledPropertyDescriptor(Object object) {
 				 null));
 	}
 
+	/**
+	 * This adds a property descriptor for the Transport MQTT Ssl Keystore Location feature.
+	 * <!-- begin-user-doc -->
+	 * <!-- end-user-doc -->
+	 * @generated NOT
+	 */
+	protected void addTransportMQTTSslKeystoreLocationPropertyDescriptor(Object object) {
```

Comment:
There seems to be an issue in formatting.

Reply:
This code is auto-generated. Need to figure out some other way to fix the issue.